test/algorithms/set_operations/difference/difference.cpp

```diff
@@ -345,17 +345,18 @@ void test_all()
 #endif
 
     {
-        //geos_2_a #area expected: 138.69238279999999008 detected: 138.53125 type: f
-        //geos_2_b #area expected: 211.859375 detected: 210.53125 type: f
+        ut_settings settings;
+        settings.set_test_validity(BG_IF_RESCALED(false, true));
 
-        // MSVC 14 expects 138.69214 and 211.85913: increase percentage
         // Output polygons for sym difference might be combined
+        expectation_limits a{138.5312, 138.6924};
```

Comment:
where are these numbers come from?

Reply:
From latest tests.
Especially if there are nearly collinear segments, the calculated areas can differ.
If they differ I always check them visually (with SVG's)